### PR TITLE
Test

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: CalculatorWithTests.Test/CalculatorTests.cs [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+namespace CalculatorWithTests.Tests
+{
+    public class CalculatorTests
+    {
+        [Theory]
+        [InlineData(1, 1, 2)]
+        [InlineData(-5, -2, -7)]
+        [InlineData(0, 0, 0)]
+        [InlineData(null, 5, 0)]
```
Comment (chipelmer) [inline CalculatorWithTests.Test/CalculatorTests.cs:13]: `null` for an integer parameter?  If you want this ability, have a look at nullable types, such as `int?`, `bool?`, etc.

File: CalculatorWithTests.Test/CalculatorTests.cs [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+            // Assert
+            Assert.Equal(expected, actual);
+        }//Divide
+
+        [Theory]
+        [InlineData(1, 0)]
+        [InlineData(-5, null)]
+        [InlineData(0, 0)]
+        [InlineData(null, 0)]
+        public void Factorial(int number, int expected)
```
Comment (chipelmer) [inline CalculatorWithTests.Test/CalculatorTests.cs:82]: This method needs a test case or two that checks the basic functionality of Factorial, such as 4! = 24, 5! = 120, 10! = 3628800, etc., instead of only the edge cases.
Also, 0! = 1

**Review by chipelmer — APPROVED** [review]
Good work, have a look at the Factorial test cases if you get a chance
